mcstatus/mc_server.py

```diff
@@ -0,0 +1,157 @@
+"""Module for storing `MCServerResponse` object."""
```

Comment:
We generally don't use file-wide docstrings, file name and the file content is usually indicative enough of what that file is for.

Reply:
I thinking about rename it to `models.py`, because all what it actually contain - just dataclasses with `build` methods.

Reply:
I still think the name should include that the file is holding classes about status responses, it makes it very intuitive to look into that file if someone is looking for them, `models` is a bit too vague it may make sense if we expected that file to hold more things over time, but I'd rather keep all status response classes together in one single file just for them, so I think `status_response.py` or something similar to that does make sense here

Reply:
Change in https://github.com/py-mine/mcstatus/pull/306/commits/4ab3585c1494d29600fdb786afa95b5732e14c61.

Reply:
> Change in [4ab3585](https://github.com/py-mine/mcstatus/commit/4ab3585c1494d29600fdb786afa95b5732e14c61).

That commit did address the rename, but don't forget to also remove the file-wide docstirng.

Reply:
Removed in 01610e977d9add9a133bb08d77dbb9d60a3bce76.